flycheck.el

```diff
@@ -3872,11 +3872,13 @@ The following PROPERTIES constitute an error level:
      property for error level categories is `priority', to
      influence the stacking of multiple error level overlays.
 
-`:fringe-bitmap BITMAP'
-     A fringe bitmap symbol denoting the bitmap to use for fringe
-     indicators for this level.  See Info node `(elisp)Fringe
-     Bitmaps' for more information about fringe bitmaps,
-     including a list of built-in fringe bitmaps.
+`:fringe-bitmaps BITMAPS'
```

Comment:
Let's keep the keyword as `:fringe-bitmap`, since other code out there depends on this API.   Also, we should continue handling the non-cons case.  Maybe something like this: *A fringe bitmap symbol denoting the bitmap to use for fringe indicators for this level, or a cons of two bitmaps (one for narrow fringes and one for wide fringes).*